build/ios/mojom/mojom_objc_generator.py

```diff
@@ -270,7 +270,8 @@ class BaseDictionaryValueMojoTypemap(MojoTypemap):
     @staticmethod
     def IsMojoType(kind):
         return (mojom.IsStructKind(kind) and
-                kind.qualified_name == 'mojo_base.mojom.DictionaryValue')
+                (kind.qualified_name ==
+                 'mojo_base.mojom.DeprecatedDictionaryValue'))
```

Comment:
Are there any usages of DictionaryValue still? I assume if so then this will no longer work for those